docs/stencil-docs/developing-further/stored-credit-card-management.mdx

```diff
@@ -84,6 +84,110 @@ curl -O https://raw.githubusercontent.com/leeBigCommerce/cornerstone/54f5681a6a1
 ## Update `en.json`
 At minimum, a translation for the `en` needs to be added to `lang/en.json: [Pull Request #1603](https://github.com/bigcommerce/cornerstone/pull/1603/files?file-filters%5B%5D=.html&file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-b0d4c1fc9d8d2a5a213b27a72cf6c9fe). Add translations for other locales as needed.
 
+## Manage stored account payments
+
+After adding a stored payment method, shoppers can use the Storefront account payments microapp to manage their stored payment instruments. The Storefront account payments provide a payment method form with billing address fields. Currently, the microapp works only with Ayden.

```

Comment:
* Spelling on `Adyen`
* can we link [Storefront account payments microapp]() to somewhere in KB?

Reply:
I added this callout in [KB-14556](https://bigcommercecloud.atlassian.net/browse/KB-14556)